lib/webapp.js

```diff
@@ -710,6 +728,26 @@ const staticHandlers = {
     return router;
   },
 
+  swagger(productCode) {
+    const router = express.Router();
+    const agentSwaggerPlugin = zluxUtil.agentSwaggerPlugin;
+    const agentSwaggerBaseUrl = `${zLuxUrl.makePluginURL(productCode, agentSwaggerPlugin)}/catalogs/swagger`;
+    const zluxSwaggerBaseUrl = `${zLuxUrl.makePluginURL(productCode, 'org.zowe.configjs')}/catalogs/swagger`;
+    router.get('/api-docs/agent', function(req, res, next) {
+      res.redirect(`${agentSwaggerBaseUrl}`);
+    });
+
+    router.get('/api-docs/agent/:api', function(req, res, next) {
+      res.redirect(`${agentSwaggerBaseUrl}/${req.params.api || ''}`);
+    });
+
+    router.get(`/api-docs/${productCode}`, function(req, res, next) {
+      res.redirect(`${zluxSwaggerBaseUrl}`);
```

Comment:
therefore instead of doing this, have it just serve server-plugins-api.yaml

Reply:
yes modified, its serving combined one now